### DISCUSSION TITLE
Portal Engine creation [ERROR]

### DISCUSSION THREAD

**Discussion by Yuki1602** [pimcore/pimcore#10818] [Q&A / Support] [answered]
![image](https://user-images.githubusercontent.com/61821237/141423868-af1ae73a-aa37-46b5-adea-be1946d4b50b.png)
this is the error i am getting when creating a new portal can anyone explain ?

**Comment by blankse** [comment, marked as answer]
You need the "COM and .Net" PHP Module on windows.
See: https://www.php.net/manual/en/book.com.php
